src/main/java/net/minecraftforge/fml/common/network/FMLOutboundHandler.java

```diff
@@ -231,6 +236,71 @@ public void validateArgs(Object args)
                 return builder.build();
             }
         },
+        /**
+         * The packet is sent to all players that are watching the Chunk containing the supplied {@link TargetPoint}.
+         * The {@code range} field of the {@link TargetPoint} is ignored.
+         */
+        TRACKING_POINT(Sets.immutableEnumSet(Side.SERVER))
+        {
+            @Override
+            public void validateArgs(Object args)
+            {
+                if (!(args instanceof TargetPoint))
+                {
+                    throw new RuntimeException("TRACKING_POINT expects a TargetPoint argument");
+                }
+            }
+
+            @Nullable
+            @Override
+            public List<NetworkDispatcher> selectNetworks(Object args, ChannelHandlerContext context, FMLProxyPacket packet)
+            {
+                TargetPoint tp = (TargetPoint)args;
+                ImmutableList.Builder<NetworkDispatcher> builder = ImmutableList.builder();
+                PlayerChunkMap chunkMap = FMLCommonHandler.instance().getMinecraftServerInstance().getWorld(tp.dimension).getPlayerChunkMap();
```

Comment:
Use `DimensionManager.getWorld` here (and null-check the return). We don't want to load the dimension here - if it's not already loaded, it's devoid of players anyway.